index.js

```diff
@@ -1,232 +0,0 @@
-    confirmation === "l" ||
-    confirmation === "y" ||
-    confirmation === "n"
-  ) {
-    //Answer Lie Cheat Detector
-    if (previousGuess == secretNumber && confirmation !== "y") {
-      console.log(
-        "My friend told me you cheated, I don't want to play anymore!"
-      );
-      process.exit();
-    }
-
-    //Computer Victory
-    if (confirmation === "y") {
-      console.log("I Win!");
-      process.exit();
-    }
-
-    //The Computer Guessed Wrong
-    else if (confirmation === "n") {
-      confirmation = await ask(
-        "If your number is higher, press (h), if it is lower, press (l)"
-      );
-    }
-
-    //Modify the Low End
-    if (confirmation === "h") {
-      rangeMinimum = previousGuess;
-    }
-
-    //Range Bust Cheat Detector
-    if (
-      confirmation === "l" &&
-      previousGuess < secretNumber
-    ) {
-      console.log(
-        "My friend told me you cheated, I don't want to play anymore!"
-      );
-      process.exit();
-    }
-
-    //Range Bust Cheat Detector
-    if (
-      confirmation === "h" &&
-      previousGuess > secretNumber
-    ) {
-      console.log(
-        "My friend told me you cheated, I don't want to play anymore!"
-      );
-      process.exit();
-    }
-    
-    // Modify the High End
-    if (confirmation === "l") {
-      rangeMaximum = previousGuess;
-    }
-
-    //Modified Answer for Large Numbers
-    if ((rangeMaximum >= 4 && confirmation === "h") || confirmation === "l") {
-      confirmation = await ask(
-        `Is your number ${middleValue(
-          rangeMinimum,
-          rangeMaximum
-        )}?, (y) for yes, or is it higher (h) or lower (l)?`
-      );
-      } else if (
-        (rangeMaximum < 4 && confirmation === "h") ||
-        confirmation === "l"
-      ) {
-        previousGuess += 1;
-        confirmation = await ask(
-          `Is your number ${previousGuess}?, (y) for yes, or is it higher (h)?`
-        );
-    }
-
-    //Stores New Guess for Large Range Adjustment
-    if (rangeMaximum >= 4) {
-      previousGuess = middleValue(rangeMinimum, rangeMaximum);
-    }
-    
-    //Stores New Guess for Small Range Adjustment
-    else if (rangeMaximum < 4 && secretNumber < 4) {
-      previousGuess = randomNumber(rangeMinimum, rangeMaximum);
-    }
-  }
-}
-
-//Game Sequence Bundle
-async function gameChoiceLoop() {
-  //Pick a Game
-  let loopConfirmation = await ask(
-    "Hi, I'd like to play a guessing game.\nIf you would like to guess press (m) as in 'me'.\nIf you would like me to guess press (c) for computer.\nYou can also quit by pressing (q)."
-  );
-  while (
-    loopConfirmation === "m" ||
-    loopConfirmation === "c" ||
-    loopConfirmation === "q"
-  ) {
-    loopConfirmation = await ask(
-      "To confirm press again, though you can change your mind..."
-    );
-
-    if (loopConfirmation === "c") {
-      computerGuessGame();
-    }
-    if (loopConfirmation === "m") {
-      humanGuessGame();
-    }
```

Comment:
While testing your game and attempting to break it I was left hanging in the terminal when I added a space before the ['m', 'c', 'q'] was input. Sanitizing user input is something that can be useful. You could even create a function to sanitize fresh user inputs as they are taken in. In this example you could .trim() your loopConfirmation variable at some point to remove any extra spaces on each end. Just something to think about!